interface-definitions/interfaces_wireless.xml.in

```diff
@@ -535,6 +547,18 @@
                       </constraint>
                     </properties>
                   </leafNode>
+                  <leafNode name="coding-scheme">
+                    <properties>
+                      <help>Spacial Stream and Modulation Coding Scheme settings</help>
+                      <valueHelp>
+                        <format>u32:0-3</format>
```

Comment:
It would be nice from a CLI completion help point of view do have short and defined definitions like:

```xml
<valueHelp>
  <format>u32:0</format>
  <description>HE-MCS 0-7</description>
</valueHelp>
<valueHelp>
  <format>u32:1</format>
  <description>1 = HE-MCS 0-9</description>
</valueHelp>
<valueHelp>
  <format>u32:2</format>
  <description>2 = HE-MCS 0-11</description>
</valueHelp>
<valueHelp>
  <format>u32:3</format>
  <description>not supported</description>
</valueHelp>
```

Reply:
Thank you very much for this hint! That'll be much better, indeed.